shared/src/main/scala/mlscript/codegen/Scope.scala

```diff
@@ -296,6 +301,19 @@ class Scope(name: Str, enclosing: Opt[Scope]) {
     symbol
   }
 
+  def declareNewClassMember(lexicalName: Str, isByvalueRec: Option[Boolean], isLam: Boolean): NewClassMemberSymbol = {
```

Comment:
What's the logic of this? Class members should have fixed names determined by users and it is never okay to rename them. So what does this do, and why isn't that documented?

Reply:
If the renaming is not expected, I think we can simply remove the runtime name allocation and keep the runtime name the same as the lexical name in class member symbols.

Reply:
You understand why we can't rename here, right? They're not like local variables. Class members are exported and will be used by other modules, which won't know about any renaming.

Do you do other incorrect renaming of this kind anywhere else?

Reply:
So should we also need to remove code like this:
```scala
val runtimeName = allocateRuntimeName(lexicalName)
val symbol = MixinSymbol(lexicalName, runtimeName, params.sorted, base, methods, ctor)
```
and just use the lexical name?

Reply:
What do you think?

Reply:
The names of classes/mixins/modules are also exported. aren't they?

Reply:
Of course. How could they not be?

Reply:
OK! I will then remove the renaming of class/mixin/module name and keep the runtime name the same as the lexical name.